Tests/Functional/Controller/Fixtures/Database/FrontEndEditorController/TeasAssignedToUser.csv

```diff
@@ -0,0 +1,5 @@
+"tx_tea_domain_model_tea"
+,"uid","pid","title","owner"
+,1,2,"Godesberger Burgtee",1
+,2,2,"Oolong",2
+,3,2,"Sencha",0
```

Comment:
Now that we have 3 separate tests for the different cases, we should also have 3 different fixtures:

- `TeaOwnedByLoggedInUser`
- `TeaOwnedByOtherUser`
- `TeaWithoutOwner`

Reply:
I'm not a fan of that, as that would split the UIDs into three different files, which would make it hard to have an overview of the available UIDs and what they mean.